src/Domain/UseCase/GetAuthMethods/GetAuthMethodsResponse.ts

```diff
@@ -0,0 +1,15 @@
+import { Uuid } from '../../Uuid/Uuid'
+
+export type GetAuthMethodsResponse = {
+  success: true,
+  methods: {
+    // PR note: or a password could always be implied
+    password: true,
```

Comment:
If this value is always going to be true then we should not have it at all. Implicit seems better.